policyengine_us/parameters/gov/states/hi/tax/income/alternative_tax/max_amount.yaml

```diff
@@ -0,0 +1,29 @@
+description: Hawaii caps the alternative tax on capital gains at this amount, based on filing status.
+metadata:
+  label: Hawaii alternative tax on capital gains max amount
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: Hawaii Resident Income Tax Instructions N-11 Rev.2022 
+    href: https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=33
+  - title: Hawaii 2022 Individual Income Tax Return N-11 From Rev.2022-line 27
```

Comment:
remove this reference as the parameter value (24,000/36,000/48,000) please check all the references to make sure that they point to the parameter in all yaml files